### PR TITLE
PPS-1223: update/remove dependencies

### PR DESCRIPTION
**Pull request by craigrbarnes**
Link to JIRA ticket if there is one:  https://ctds-planx.atlassian.net/browse/PPS-1223

### Dependency updates

Update the versions below to resolve synk High issues

* NextJS 14.2.5
* recharts 2.12.7
* remove next-images

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{
+ {
```
Comment (mfshao) [inline package.json:1]: I think this change is unintentional?

**Review by mfshao — APPROVED** [review]
lgtm except for the extra whitespace

File: package.json (hunk 1 above)
Comment (craigrbarnes) [reply package.json:1]: yes. I will deal with it.

Comment (craigrbarnes) [reply package.json]: Removed

**Review by mfshao — APPROVED** [review]

**Pull request merged**